install/tests/Makefile

```
@@ -53,8 +53,8 @@ gcp-kubeconfig:
 ## azure-kubeconfig: Get the kubeconfig configuration for Azure AKS
 azure-kubeconfig:
 	export KUBECONFIG=${KUBECONFIG} && \
-	export resource=$$(echo "$$TF_VAR_TEST_ID" | sed "s/[\\W\\-]//") && \
-	az aks get-credentials --name gitpod-test-nor-primary-$$resource --resource-group gitpod-test-nor-$$resource --file ${KUBECONFIG} || echo "No cluster present"
+	export resource=$$(echo "$$TF_VAR_TEST_ID" | sed "s/[\\W\\-]//g") && \
+	az aks get-credentials --name test-cluster-$$resource --resource-group sh-test-$$resource --file ${KUBECONFIG} || echo "No cluster present"
```

Can you use the `terraform output` command instead? Something like...

```suggestion
	az aks get-credentials --name $(terraform output -json <name> | jq -r '.') --resource-group $(terraform output -json <name> | jq -r '.') --file ${KUBECONFIG} || echo "No cluster present"
```

---

This would involve changing the AKS terraform module output too. Do you mind if I take this up on a follow-up? I have to refactor the AKS module soon once the reference architecture is out.